dojo_app.py

```diff
@@ -0,0 +1,115 @@
+"""
+This Application automatically allocated spaces to people at random
+Usage:
+    dojo_app.py create_room <room_type> <room_name>...
+    dojo_app.py add_person <first_name> <last_name> <FELLOW/STAFF> [<wants_accomodation>]
+    dojo_app.py print_room <room_name>
+    dojo_app.py print_allocations [--o=filename]
+    dojo_app.py print_unallocated [--o=filename]
+    dojo_app.py reallocate_person <person_identifier> <new_room_name>​
+    dojo_app.py load_people <text_file>
+    dojo_app.py -h | --help
+    dojo_app.py -V | --version
+    dojo_app.py -i | --interactive
+
+Options:
+    -o                      Outputs the result into a text file.
+    -h, --help              Show message and exit.
+    -V, --version           Show the version.
+    -i, --interactive       interactive.
+
+
+"""
+import sys
+import cmd
+from docopt import docopt, DocoptExit
+from models.dojo import Dojo
+
+
+def docopt_cmd(func):
+    """
+    This decorator is used to simplify the try/except block and pass the result
+    of the docopt parsing to the called action.
+    """
+
+    def fn(self, arg):
+        try:
+            opt = docopt(fn.__doc__, arg)
+
+        except DocoptExit as e:
+            # The DocoptExit is thrown when the args do not match.
+            # We print a message to the user and the usage block.
+
+            print('Invalid Command!')
+            print(e)
+            return
+
+        except SystemExit:
+            # The SystemExit exception prints the usage for --help
+            # We do not need to do the print here.
+
+            return
+
+        return func(self, opt)
+
+    fn.__name__ = func.__name__
+    fn.__doc__ = func.__doc__
+    fn.__dict__.update(func.__dict__)
+    return fn
+
+
+class DojoRoom(cmd.Cmd):
+    """Welcome to the Dojo Room App!"""
+    prompt = '(Dojo) '
+    file = None
```

Comment:
`file` is a reserved word in Python and you shouldn't use it as a variable name. Besides I don't see where you are using this in the code. Is it needed at all ?